setup.py

```diff
@@ -32,14 +32,15 @@
 
 
     error = """
-Jupyter_Console 6.2+ supports Python 3.6 and above.
+Jupyter_Console 6.2+ supports Python 3.7 and above.
 When using Python 2.7, please install and older version of Jupyter Console
 Python 3.3 and 3.4 were supported up to Jupyter Console 5.x.
 Python 3.5 was supported up to Jupyter Console 6.1.0.
+Python 3.6 was supported up to Jupyter Console 6.4.3.
```

Comment:
Does this change need a minor version bump?
Should this note say 6.5?

Reply:
Either way this note is correct, right?